rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/EnricherConfigSummary.java

```diff
@@ -20,60 +20,24 @@
 
 import java.net.URI;
 import java.util.Map;
-import java.util.Objects;
 
 import org.apache.brooklyn.config.ConfigKey;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.google.common.collect.ImmutableMap;
-
-public class EnricherConfigSummary extends ConfigSummary {
+// TODO remove? this class has no value over its super
```

Comment:
Agreed - I think we can safely delete it. The json returned by `AdjunctConfigSummary` will be identical to `EnricherConfigSummary` etc. But we can do that in a future PR.

Not sure what the difference is between `EntityConfigSummary` and `AdjunctConfigSummary` (it has "pinned" and "constraints"). Adjuncts can have constraints, I believe. For "pinned", the `CatalogConfig` javadoc says "a pinned configuration means that the config key will always be displayed when presenting as editable in the catalog", so I think that also applies to adjuncts (well, most adjuncts and eventually all).

Reply:
agree, there is no reason to have more than one `ConfigSummary`